mesonbuild/mtest.py

```diff
@@ -331,11 +331,12 @@ def make_exitcode(cls, test: 'TestSerialisation', test_env: typing.Dict[str, str
             res = TestResult.EXPECTEDFAIL if bool(returncode) else TestResult.UNEXPECTEDPASS
         else:
             res = TestResult.FAIL if bool(returncode) else TestResult.OK
-        return cls(test, test_env, res, returncode, duration, stdo, stde, cmd)
+        return cls(test, test_env, res, returncode, starttime, duration, stdo, stde, cmd)
 
     @classmethod
     def make_tap(cls, test: 'TestSerialisation', test_env: typing.Dict[str, str],
-                 returncode: int, duration: float, stdo: str, stde: str,
+                 returncode: int, starttime:float, duration: float,
```

Comment:
__[Flake8]__ missing whitespace after ':' ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6142#issue-4807815))

<sub>posted by Sider</sub>

Reply:
@ignatenkobrain Yes, just saw that and pushed a commit seconds ago. Thank you for the heads-up.